public_html/administrator/components/com_jed/views/review/view.html.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * @package    JED
+ *
+ * @copyright  Copyright (C) 2005 - 2020 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+use Joomla\Registry\Registry;
+use Joomla\CMS\Factory;
+use Joomla\CMS\Helper\ContentHelper;
+use Joomla\CMS\MVC\View\HtmlView;
+use Joomla\CMS\Toolbar\ToolbarHelper;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Object\CMSObject;
+
+/**
+ * View to edit a review.
+ *
+ * @since  4.0.0
+ */
+class JedViewReview extends HtmlView
+{
+	/**
+	 * Form with settings
+	 *
+	 * @var    form
+	 *
+	 * @since  4.0.0
+	 */
+	protected $form;
+
+	/**
+	 * The item data.
+	 *
+	 * @var   CMSObject
+	 *
+	 * @since 4.0.0
+	 */
+	protected $item;
+
+	/**
+	 * The model state.
+	 *
+	 * @var   Registry
+	 *
+	 * @since 4.0.0
+	 */
+	protected $state;
+
+	/**
+	 * Display the view
+	 *
+	 * @param   string  $tpl  The name of the template file to parse; automatically searches through the template paths.
+	 *
+	 * @return  void
+	 *
+	 * @throws  Exception
+	 *
+	 * @since   4.0.0
+	 */
+	public function display($tpl = null)
+	{
+		/** @var JedModelReview $model */
+		$model       = $this->getModel();
+		$this->state = $model->getState();
+		$this->item  = $model->getItem();
+		$this->form  = $model->getForm();
+		$errors      = $model->getErrors();
+
+		if ($errors && count($errors))
+		{
+			throw new RuntimeException(implode("\n", $errors), 500);
+		}
+
+		$this->addToolbar();
+		parent::display($tpl);
```

Comment:
Just an empty line before the last statement and add `return` before it so it becomes:
```
return parent::display($tpl);
```